src/components/MovieDetails/MovieDetails.jsx

```diff
@@ -0,0 +1,49 @@
+import './MovieDetails.css'
+
+function MovieDetails(props) {
+
+  function formatDate(inputDate) {
+    const options = { month: 'long', day: 'numeric', year: 'numeric', timeZone: 'UTC' };
```

Comment:
Dates in JS are a pain!! Considering using a date library to help with this; moment.js and day.js help solve these problems with very little effort